src/shared/components/layouts/UniProtFooter.tsx

```diff
@@ -302,14 +302,14 @@ const FooterShortcuts = () => (
   </ul>
 );
 
-const FooterContactAndElixir = () => (
-  <div className="contact-elixir">
+export const Contact = () => (
```

Comment:
Wondering whether this might be better as its own component rather than in footer? Might need to put it in other places later.

Reply:
Done